src/components/atoms/DateTimePicker/Component.tsx

```diff
@@ -147,7 +147,7 @@ const DateTimePicker: FunctionComponent<DateTimePickerProps> = forwardRef<
             <Icon
               color={defaultTheme?.colors?.grey3}
               name="calendar"
-              type="antdesign"
+              type="AntDesign"
```

Comment:
Can we use this enum instead of using plain string

https://github.com/Jitera/jitera-rn-ui-library/blob/71771f2cd84c34442d59565ede6332a842e07bf2/src/components/atoms/Icon/Component.tsx#L8-L23

Reply:
sure